tests/unit_tests/test_surface_composite.py

```diff
@@ -393,3 +394,64 @@ def test_polygon():
                           [6.88, 3.02]])
     with pytest.raises(ValueError):
         openmc.model.Polygon(rz_points)
+
+
+@pytest.mark.parametrize("axis", ["x", "y", "z"])
```

Comment:
add test to catch errors after implementing the `distances` setter

Reply:
:heavy_check_mark: